src/test.ts

```diff
@@ -104,9 +109,32 @@ export function runTest(testDirectory: string, rulesDirectory?: string | string[
             };
         });
 
+        // test against fixed files
+        let fixedFileText: string;
+        let newFileText: string;
+        try {
+            const fixedFile = fileToLint.replace(/\.lint$/, FIXES_FILE_EXTENSION);
+            const stat = fs.statSync(fixedFile);
+            if (stat.isFile()) {
+                fixedFileText = fs.readFileSync(fixedFile, "utf8");
+                const fixes: Fix[] = [];
```

Comment:
Believe you could simplify this a little:
`const fixes = failures.filter((f) => f.hasFix()).map((f) => f.getFix());`